README.md

```diff
@@ -55,6 +55,12 @@ In general follow this [Javascript Styleguide](https://google.github.io/stylegui
 docker-compose -f "docker-compose.yml" up -d --build
 ```
 
+If this command doesn't work, run 
```

Comment:
you should probably mentioned something like "if you are running Mac" then

If this command doesn't work, run 

```bash
PORT=9119 docker-compose up
```

If you are running on Linux like OS, then do:
```bash
docker-compose up -e PORT=9119
```